easybuild/easyblocks/b/boost.py

```diff
@@ -120,8 +121,12 @@ def configure_step(self):
             raise EasyBuildError("When enabling building boost_mpi, also enable the 'usempi' toolchain option.")
 
         # create build directory (Boost doesn't like being built in source dir)
-        self.objdir = os.path.join(self.builddir, 'obj')
-        mkdir(self.objdir)
+        try:
+            self.objdir = os.path.join(self.builddir, 'obj')
+            mkdir(self.objdir)
```

Comment:
The mkdir from easybuild filetools already handle this.

Reply:
@valtandor what @akesandgren is saying is that the error handling you added is already present the `mkdir` function.
in view of that, I think we can close this PR, do you agree?